### PR TITLE
Fix log to file not work

### PR DESCRIPTION
**Pull request by jiaoew1991**
Signed-off-by: Enwei Jiao <enwei.jiao@zilliz.com>
issue: #20535

### CODE REVIEW

File: cmd/roles/roles.go [hunk 1]
```diff
@@ -188,11 +188,7 @@ func (mr *MilvusRoles) setupLogger() {
 	roleName := paramtable.GetRole()
 	rootPath := logConfig.File.RootPath
 	if rootPath != "" {
-		if id < 0 {
-			logConfig.File.Filename = path.Join(rootPath, roleName+".log")
-		} else {
-			logConfig.File.Filename = path.Join(rootPath, roleName+"-"+strconv.FormatInt(id, 10)+".log")
-		}
+		logConfig.File.Filename = path.Join(rootPath, roleName+"-"+strconv.FormatInt(id, 10)+".log")
```
Comment (XuanYang-cn) [inline cmd/roles/roles.go:191]: maybe `fmt.Sprintf("%s-%d.log", roleName, id)` ?

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]

File: cmd/roles/roles.go (hunk 1 above)
Comment (jiaoew1991) [reply cmd/roles/roles.go:191]: it seems better 🤔

**Review by congqixia — APPROVED** [review]
/kind bug
/lgtm

**Pull request merged**